### PR TITLE
机械臂资料添加

### PR DESCRIPTION
**Pull request by pengziliang**

### CODE REVIEW

File: _posts/2024-04-09-myAGV.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+tags:
+- 'arm'
+- 'education'
+- 'elephantrobotics'
+- 'manipulator'
+- 'research'
+- 'myAGV'
+twitter_text: Checkout the myAGV
+website: https://www.elephantrobotics.com/en/myAGV-en/
+wiki_homepage: http://wiki.ros.org/Robots/myAGV
```
Comment (tfoote) [inline _posts/2024-04-09-myAGV.md]: Please point the wiki_homepage to a link where there's ROS developer documentation on how to interface to the devices. For this and all below. I only checked a few that didn't seem to exist on the wiki as currently linked.

**Review by tfoote — CHANGES_REQUESTED** [review]
The changes look fine from reading. Do you know why this isn't running the CI actions?

File: _posts/2024-04-09-myAGV.md (hunk 1 above)
Comment (pengziliang) [reply _posts/2024-04-09-myAGV.md]: Hello, what issue are you encountering? Is it that the wiki_homepage link cannot be redirected?

Comment (tfoote) [reply _posts/2024-04-09-myAGV.md]: That link: http://wiki.ros.org/Robots/myAGV has no content. 

image

Comment (tfoote) [reply _posts/2024-04-09-myAGV.md]: I see you've fixed the links now. Closing this

**Review by tfoote — APPROVED** [review]
Thanks for the updates. It's now passing CI too.

**Pull request merged**